### ISSUE TITLE
Alternative use for meat instead of increasing grab limit

### ISSUE THREAD

**Issue by TisFoolish** [naelstrof/KoboldKare#89]
Currently grabbing is a pain if you're 'bolds are all children of meat eaters as you can very easily inadvertently grab several items along with the one item you want with no easy way to seperate them.

My suggestion is that when you eat meat it lets your child hold +1 item in their belly. So if you need to move a lot of items you just vore them. This would be keeping on brand for the kobolds of KK and keeping with the theme of this being a silly, furry smut game.

As for how it comes out, it's probably best to have a toggle for users if they want it to come out of their mouth or ass.

**Comment by naelstrof** [comment]
Haha, yeah this was legitimately how the vore mechanic would've made it in if I had more dev time. I'd love to add it, but I cannot promise it.

Hopefully I can at least make the grab upgrade not feel like such a side-grade by adding a grab limit toggle.

**Comment by TisFoolish** [comment]
Instead of a toggle, what about making it so that the user can use the
scroll wheel to increase/decrease the limit whenever they grab something?
Eg I grab an egg and scroll up to increase the limit, then scroll down to
drop the extra stuff as the limit goes down.

It's not as horny as object vore, but it solves the issue

**Comment by naelstrof** [comment]
Yeah that's a good idea, I need to make sure it translates to Xbox/Steam deck controls naturally, but it should be doable.

**Comment by TisFoolish** [comment]
Yeah, I was trying to figure out some way to get a controller layout to
work with it as that's how I play.

I can't remember if right trigger is used for anything while grabbing
stuff, but if not what about right trigger and one of the analogue sticks?

**Comment by TisFoolish** [comment]
For the Deck you could natively support the track pads for this

**Comment by Slayerhero90** [comment]
Seeing that it's come up here, I'd like to post here to voice my support for vore/object-swallowing mechanics in the game. My first thought after my first session with this game the other night was, "This game has all the submechanics that would be required for vore, from the large insertion handling, to the belly capacity booster, to the way positioning for sex is handled by having cocks seek holes. If only!"

Then to see that it was planned but cut for time is a bittersweet thing. I did have my own thoughts as to how vore could be implemented using existing systems and what it could be good for, if you'd like me to post those in a separate issue.
